### PR TITLE
Motivation

### PR DESCRIPTION
**Pull request by sjanssen2**
According to the [QIIME 2 forum](https://forum.qiime2.org/t/about-the-community-plugins-category/1617) a community plugin (and our plugin starts as a community plugin) "should include a description of why a reader would be interested in the plugin".
I added two strong arguments for SEPP.

### CODE REVIEW

**Comment by sjanssen2** [comment]
@antgonza can volunteer for a quick review?

File: README.md [hunk 1]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
     conda install -c https://conda.anaconda.org/biocore q2-fragment-insertion
     qiime dev refresh-cache
 
+## Why you should prefer fragment insertion aka SEPP over de-novo trees
```
Comment (antgonza) [inline README.md]: aka -> AKA

File: README.md [hunk 2]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
...
+
+### Fragment insertion avoids artificially long outgroup branches that would lead to exaggerated separation in beta diversity.
+
+<img src="Example/denovoArtifacts.png">
+
+Beta diversity was computed for all 599 samples of this study (manuscript in preparation) on the non rarefied deblur table with 4,727 sOTUs total as unweighted unifrac distance with three alternative phylogenetic trees:
```
Comment (antgonza) [inline README.md]: It will be good to add the study ids and links to qiita here.

File: README.md [hunk 3]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
...
+  B) Mean path length from root to tips in the denovo tree is 0.94, while the lowest common ancestor for the three *Methanobrevibacter* sOTUs has an outstanding length of 1.43. Manually shortening the grandparent's branch length from 0.82 to 0.4 re-unites clusters.
+
+  C) Inserting denovo fragments into a well curated reference phylogeny via the fragment insertion plugin also resolves cluster separation but does not require any manual manipulation.
+
+### Fragment insertion enables meta-analyses across different variable 16S regions and fragment length.
+
+<img src="Example/metaanalysis.png">
+
+Meta-analyses of two microbiome studies with heterogeneous variable 16S regions. 
+Both studies sampled the same three body products: [Study 'Family'](https://qiita.ucsd.edu/study/description/797) contains 854 human and 217 dog samples with 37,181 sOTUs of the first 128nt from V2 [[Song et al.]](http://dx.doi.org/10.7554/eLife.00458), while [study 'Yanomani'](https://qiita.ucsd.edu/study/description/10052) comprises 66 samples of uncontacted Amerindians in Venezuela with 17,249 sOTUs of the first 150nt of V4 [[Clemente et al.]](http://dx.doi.org/10.1126/sciadv.1500183).
```
Comment (antgonza) [inline README.md]: Some of these references have double []

File: README.md [hunk 4]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
...
+Beta diversity was computed on one non rarefied deblur table combining both studies as unweighted unifrac distance.
+
+  A) De-novo tree construction via aligning sOTU sequences with mafft and reconstructing the phylogeny via fasttree results in strong artifacts in PcoA (black arrow), because separation is extremely driven by variable region.
```
Comment (antgonza) [inline README.md]: PcoA -> PCoA, should at least once add Principal Coordinate Analysis (PCoA)

File: README.md [hunk 5]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
...
+
+  B) Inserting sOTU sequences from different regions into the same backbone tree as SEPP does, results in a phylogeny that correctly separates samples in PcoA space by body product. Study effect is rather marginal.
+
+  C) The 'Yanomani' samples were additionally profiled targeting the V2 region, thus we could insert the resulting 6,604 sOTUs together with the 'Family' study samples into the same phylogeny as a positive control. Separation is indeed driven by "body product" and not by study.
+
 ## Important
```
Comment (antgonza) [inline README.md]: Should this be Notes vs. Important?

File: README.md [hunk 6]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
...
+
+  C) Inserting denovo fragments into a well curated reference phylogeny via the fragment insertion plugin also resolves cluster separation but does not require any manual manipulation.
+
+### Fragment insertion enables meta-analyses across different variable 16S regions and fragment length.
+
+<img src="Example/metaanalysis.png">
+
+Meta-analyses of two microbiome studies with heterogeneous variable 16S regions. 
+Both studies sampled the same three body products: [Study 'Family'](https://qiita.ucsd.edu/study/description/797) contains 854 human and 217 dog samples with 37,181 sOTUs of the first 128nt from V2 [[Song et al.]](http://dx.doi.org/10.7554/eLife.00458), while [study 'Yanomani'](https://qiita.ucsd.edu/study/description/10052) comprises 66 samples of uncontacted Amerindians in Venezuela with 17,249 sOTUs of the first 150nt of V4 [[Clemente et al.]](http://dx.doi.org/10.1126/sciadv.1500183).
+Beta diversity was computed on one non rarefied deblur table combining both studies as unweighted unifrac distance.
```
Comment (wasade) [inline README.md]: why was the table not rarefied?

File: README.md (hunk 2 above)
Comment (wasade) [inline README.md]: why was the table not rarefied?

Comment (wasade) [reply README.md]: unifrac -> UniFrac

File: README.md [hunk 7]
```diff
@@ -5,6 +5,34 @@ Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), and
...
+
+  A) De-novo tree construction via aligning sOTU sequences with mafft and reconstructing the phylogeny via fasttree results in strong artifacts in PcoA (black arrow), because separation is extremely driven by variable region.
+
+  B) Inserting sOTU sequences from different regions into the same backbone tree as SEPP does, results in a phylogeny that correctly separates samples in PcoA space by body product. Study effect is rather marginal.
```
Comment (wasade) [inline README.md]: PcoA -> PCoA

Comment (wasade) [reply README.md]: "correctly separates" -> "separates samples as expected"

Comment (antgonza) [reply README.md]: I will go one further and say: separates samples by body product, as expected ... or something like that.

File: README.md (hunk 6 above)
Comment (sjanssen2) [reply README.md]: I did not want to introduce any bias by me choosing an arbitrary rarefaction depth. But when using a depth of 5870 as Serene and Zech did, the results look very similar:

raremros

File: README.md (hunk 1 above)
Comment (sjanssen2) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (sjanssen2) [reply README.md]: This study is not yet public.

File: README.md (hunk 3 above)
Comment (sjanssen2) [reply README.md]: I did so for references to articles to resemble citation style in papers.

File: README.md (hunk 4 above)
Comment (sjanssen2) [reply README.md]: good point

File: README.md (hunk 5 above)
Comment (sjanssen2) [reply README.md]: I agree

File: README.md (hunk 2 above)
Comment (sjanssen2) [reply README.md]: changed

File: README.md (hunk 7 above)
Comment (sjanssen2) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (antgonza) [reply README.md]: That's fine, I _think_. You could add a note about this ...

**Comment by wasade** [comment]
@sjanssen2 we've published multiple papers advising the use of rarefaction, and it is actively recommended in our tutorials, so it's a bit weird to make the motivation off of unrarefied data

File: README.md (hunk 3 above)
Comment (antgonza) [reply README.md]: I might be wrong but normally brackets are used with numbers and () with names, no? As you can imagine, I don't have a strong feeling about this, just wanted to be sure it was intentional.

Comment (sjanssen2) [reply README.md]: I am pretty sure both ways exist and it depend on the journal. Don't expect consistency :-)

**Comment by sjanssen2** [comment]
I switched to the rarefied version of argument 1. You might need to refresh you cache to see the updated figure.

**Pull request merged**